### PR TITLE
SCALRCORE-28980 API > Policy Engine > Enforce policy group for all cu…

### PR DESCRIPTION
**Pull request by denkl**
…rrent and future environments

### Changelog
* [x] I have updated CHANGELOG.md according to [CONTRIBUTING.md](../CONTRIBUTING.md)

### Documentation
* [ ] I have updated resource/data source documentation in [docs](../docs)

### State
* [ ] My changes affect the state
* [ ] I have included a state migration and a unit test for it

### CODE REVIEW

**Review by DayS1eeper — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `scalr_workspace`: new attribute `vcs-repo.trigger_patterns` ([#282](https://github.com/Scalr/terraform-provider-scalr/pull/282))
 
+### Changed
```
Comment (petroprotsakh) [inline CHANGELOG.md:16]: This should go to the 'Unreleased' section

File: scalr/resource_scalr_policy_group.go [hunk 2]
```diff
@@ -108,15 +108,38 @@ func resourceScalrPolicyGroup() *schema.Resource {
...
+
+	environments := make([]*scalr.Environment, 0)
+	var hasWildcard bool
+	for i, environmentID := range environmentIDs {
+		environments = append(environments, &scalr.Environment{ID: environmentID.(string)})
+
+		if environmentID.(string) == "*" {
+			hasWildcard = true
+		}
+		if i != 0 && hasWildcard {
```
Comment (petroprotsakh) [inline scalr/resource_scalr_policy_group.go:135]: So this allows the list to be `["*", "env-xxx", "env-yyy"]`?
I find the logic in [this example](https://github.com/Scalr/terraform-provider-scalr/blob/880edf59a8c17a39c0f6e0e8128c02703cf8f9b3/scalr/resource_scalr_vcs_provider.go#L233) more clear and correct.

**Review by petroprotsakh — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (denkl) [reply CHANGELOG.md]: fixed, thanks

File: scalr/resource_scalr_policy_group.go (hunk 2 above)
Comment (denkl) [reply scalr/resource_scalr_policy_group.go]: logic is correct, but changed it to be consistent with existing ones

**Review by petroprotsakh — APPROVED** [review]

**Pull request merged**